tests/components/homeassistant_yellow/test_hardware.py

```diff
@@ -13,6 +13,16 @@
 from tests.typing import WebSocketGenerator
 
 
+@pytest.fixture(autouse=True)
+def mock_async_validate_hardware_consistent():
```

Comment:
Maybe move this to `tests/components/homeassistant_yellow/conftest.py` to avoid surprises when adding tests in the future. Then override or parametrize the fixture in `tests/components/homeassistant_yellow/test_helpers.py`.